index.js

```diff
@@ -3,7 +3,7 @@ const noise2Png = require('./noise2');
 const noise3Png = require('./noise3');
 const HUE_SHIFT_INTENSITY = 4;
 const BRIGHT_GREEN = '#28FC91';
-const DARK_GREEN = '#0F2218';
+const DARK_GREEN = 'transparent';
```

Comment:
@thedonquixotic Exciting that this is your first PR! :)
Could you update it to avoid changing the value of DARK_GREEN? Instead, wherever we use DARK_GREEN, replace it with transparent.

Reply:
For instance on line 12 and 68 we should use `transparent` instead